problems/18.Roman to Integer.gs

```diff
@@ -0,0 +1,37 @@
+function romanToInt(s: string): number {
+
+    let fromRoman = new Map<string, number>([
+        ["I", 1],
+        ["V", 5],
+        ["X", 10],
+        ["L", 50],
+        ["C", 100],
+        ["D", 500],
+        ["M", 1000]
+    ]);
+
+    let sum = 0
+    for (let i = 0; i < s.length; i++) {
+        if (i < s.length - 1 && mapGet(fromRoman, s[i]) < mapGet(fromRoman, s[i + 1])) {
```

Comment:
extract mapGet(fromRoman, s[i]) and mapGet(fromRoman, s[i + 1]) to their own variable. Condition "if" is to large and it's hard to follow now...

Additionally, I would like to see you drop the i < s.length -1 part. You could achieve this by running for loop until s.length and add one more statement after for sum += mapGet(fromRoman, s[s.length - 1])

This will simplify things